var/spack/repos/builtin/packages/survey/package.py

```diff
@@ -69,7 +70,7 @@ class Survey(CMakePackage):
     depends_on("llvm-openmp@12.0.1", type=("build", "link"), when="@1.0.3:")
 
     # MPI Installation
-    depends_on("mpi", when="+mpi")
+    depends_on("mpi", type="build", when="+mpi")
```

Comment:
MPI is only needed for the build of the package?

Reply:
@tldahlgren Yes.  We just need the definitions to compile our MPI data collection code.

Reply:
@tldahlgren The source is protected (not open source), that is probably why you can't get to this branch.